CONTRIBUTING.md

```diff
@@ -58,21 +58,21 @@ When creating a pull request, please provide the following information:
 
 In TSDoc comments, always make sure to align the `-` for all `@param`s.
 
-In very rare occasions, it is acceptable to disable prettier for a specific part of the code to improve the formatting manually, e.g. for deeply nested conditional type, which are just unreadable if we let prettier format them. See  [Prettier – Ignoring Code](https://prettier.io/docs/en/ignore.html#javascript) to find out how to do this.
+In very rare occasions, it is acceptable to disable prettier for a specific part of the code to improve the formatting manually, e.g. for deeply nested conditional types, which are simply unreadable if we let prettier format them. See  [Prettier – Ignoring Code](https://prettier.io/docs/en/ignore.html#javascript) to find out how to do this.
 
 ### General Guidelines
 
-1. Try to match the source code of `foundry.js` (and other modules from 0.8.0 onwards) as closely as possible in your type definitions. In particular, the order of declaration should be exactly the same. This allows for easy side by side viewing of `foundry.js` and the type definitions, making the life of code reviewers much easier :)
-2. Try not to pollute the global namespace with custom types that are not declared by foundry itself (typedefs are ok and should be declared). Instead use a namespace named like the related class and put your custom type in there.
-3. Every class has its own file. The files are structured by class hierarchy. Declarations for code that is not a class belongs directly in `foundry` folder. For 0.8.0 we are still figuring out how to structure things with respect to the namespaces / modules that are introduced.
+1. Try to match the source code of `foundry.js` (and other modules from 0.8.0 onwards) as closely as possible in your type definitions. In particular, the order of declarations should be exactly the same. This allows for easy side by side viewing of `foundry.js` and the type definitions, making the life of code reviewers much easier :)
+2. Try not to pollute the global namespace with custom types that are not declared by foundry itself (typedefs from `foundry.js` should be declared). Instead use a namespace named like the related class and put your custom type in there.
+3. Every class has its own file. The files are structured by class hierarchy. Declarations for code that is not a class belongs directly in the `foundry` folder. For 0.8.0 we are still figuring out how to structure things with respect to the namespaces / modules that are introduced.
 4. Utility types not defined in foundry belong in `types/utils.d.ts`
 5. Augments for libraries bundled with Foundry VTT belong in their corresponding file in `types/augments`.
 6. Write tests were applicable. Not everything needs to be tested, these are just type definitions after all. But in particular for complicated type definitions it makes a lot of sense to add tests. That way you can also verify for yourself that your type definitions are actually working as intended.
```

Comment:
```suggestion
6. Write tests where applicable. Not everything needs to be tested, these are just type definitions after all. But in particular for complicated type definitions it makes a lot of sense to add tests. That way you can also verify for yourself that your type definitions are actually working as intended.
```

Reply:
If you add this change, I'll approve it as well.

Reply:
@kmoschcau done